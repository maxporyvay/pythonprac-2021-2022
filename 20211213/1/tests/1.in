[17, 45, 2, 90, 34, -6, 0, 0, 23, 87, 97, 12, -123, 45, 5, 2]