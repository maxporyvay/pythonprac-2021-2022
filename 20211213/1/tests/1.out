[-123, -6, 0, 0, 2, 2, 5, 12, 17, 23, 34, 45, 45, 87, 90, 97]
